tests/test_dataset/test_lyft_dataset.py

```diff
@@ -56,27 +58,28 @@ def test_getitem():
     assert np.allclose(pcd_rotation, pcd_rotation_expected, 1e-3)
     assert sample_idx == \
         'b98a05255ba2632e957884758cb31f0e6fcc8d3cd6ee76b6d0ba55b72f08fc54'
-    expected_points = torch.tensor([[59.1695, -1.2910, 7.0296, 0.2000],
-                                    [52.4867, -4.0315, 6.7057, 0.0000],
-                                    [52.5683, -4.2178, 6.7179, 0.0000],
+    expected_points = torch.tensor([[61.4785, -3.7393, 6.7699, 0.4001],
                                     [47.7904, -3.9887, 6.0926, 0.0000],
-                                    [59.8226, -1.5522, 6.5867, 0.4001],
-                                    [53.0842, -3.7064, 6.7811, 0.0000],
-                                    [49.9896, -4.5202, 5.8823, 0.2000],
+                                    [52.5683, -4.2178, 6.7179, 0.0000],
+                                    [52.4867, -4.0315, 6.7057, 0.0000],
                                     [59.8372, -1.7366, 6.5864, 0.4001],
-                                    [61.4597, -4.6402, 7.3340, 0.2000],
-                                    [61.4785, -3.7393, 6.7699, 0.4001],
-                                    [53.0702, -3.8868, 6.7807, 0.0000],
-                                    [59.8244, -1.3499, 6.5895, 0.4001],
+                                    [53.0842, -3.7064, 6.7811, 0.0000],
                                     [60.5549, -3.4978, 6.6578, 0.4001],
+                                    [59.1695, -1.2910, 7.0296, 0.2000],
+                                    [53.0702, -3.8868, 6.7807, 0.0000],
                                     [47.9579, -4.1648, 5.6219, 0.2000],
-                                    [61.2858, -4.2254, 7.3089, 0.2000]])
+                                    [59.8226, -1.5522, 6.5867, 0.4001],
+                                    [61.2858, -4.2254, 7.3089, 0.2000],
+                                    [49.9896, -4.5202, 5.8823, 0.2000],
+                                    [61.4597, -4.6402, 7.3340, 0.2000],
+                                    [59.8244, -1.3499, 6.5895, 0.4001]])
```

Comment:
Why does the result change here?

Reply:
Because we replace the random function from numpy to pytorch, so we get diffrent shuffled points.